modules/lvms-creating-volume-snapshots.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-creating-volume-snapshots_{context}"]
+= Creating volume snapshots
+
+You can create volume snapshots based on the available capacity of the thin pool and the over-provisioning limits.
+To create a volume snapshot, you must create a `VolumeSnapshotClass` object.
+
+.Prerequisites
+
+* You have access to {product-title} as a user with `cluster-admin` permissions.
+* You ensured that the persistent volume vlaim (PVC) is in `Bound` state. This is required for a consistent snapshot.
```

Comment:
Was just glancing, and wondering if this is a typo, and if it should say "persistent volume claim" instead of "persistent volume vlaim."

Reply:
Hi @gwynnemonahan 
Yes it should be "claim' instead of "vlaim". Thank you for pointing this out.